### PR TITLE
enforce 80-character line length

### PR DESCRIPTION
**Pull request by gmartinonQM**
# Description

Enforce 80-character line length

## Type of change

Please check options that are relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] I have read the [contributing guidelines](https://github.com/simai-ml/MAPIE/blob/master/CONTRIBUTING.rst)
- [ ] I have updated the [HISTORY.rst](https://github.com/simai-ml/MAPIE/blob/master/HISTORY.rst) and [AUTHORS.rst](https://github.com/simai-ml/MAPIE/blob/master/AUTHORS.rst) files
- [x] Linting passes successfully : `flake8 . --exclude=doc`
- [x] Typing passes successfully : `mypy mapie examples --strict`
- [x] Unit tests pass successfully : `pytest -vs --doctest-modules mapie`
- [x] Coverage is 100% : `pytest -vs --doctest-modules --cov-branch --cov=mapie --pyargs mapie`
- [x] Documentation builds successfully : `cd doc; make clean; make html`

### CODE REVIEW

File: mapie/estimators.py [hunk 1]
```diff
@@ -430,17 +506,24 @@ def fit(self, X: ArrayLike, y: ArrayLike, sample_weight: Optional[ArrayLike] = N
...
+            )
             if self.method == "naive":
                 y_pred = self.single_estimator_.predict(X)
             else:
-                cv_outputs = Parallel(n_jobs=self.n_jobs, verbose=self.verbose)(
+                outputs = Parallel(n_jobs=self.n_jobs, verbose=self.verbose)(
                     delayed(self._fit_and_predict_oof_model)(
-                        clone(estimator), X, y, train_index, val_index, k, sample_weight
+                        clone(estimator),
+                        X, y, train_index, val_index, k, sample_weight
```
Comment (vtaquet) [inline mapie/estimators.py:518]: Ne faut-il pas faire un retour à la ligne après chaque argument pour respecter la convention ?

File: mapie/estimators.py [hunk 2]
```diff
@@ -470,39 +554,60 @@ def predict(self, X: ArrayLike) -> np.ndarray:
...
+        )
         X = check_array(X, force_all_finite=False, dtype=["float64", "object"])
         alpha = self._check_alpha(self.alpha)
 
         # SHAPES:
         # (n_samples_train,) : self.residuals_
         # (n_alpha,) : alpha
         # (n_samples_test, n_alpha) : y_pred_low, y_pred_up
-        # (n_samples_test, n_samples_train) : y_pred_multi, lower_bounds, upper_bounds
+        # (n_samples_test, n_samples_train) :y_pred_multi, lower/upper_bounds
```
Comment (vtaquet) [inline mapie/estimators.py:574]: Il manque un espace après `:`

File: mapie/tests/test_estimators.py [hunk 3]
```diff
@@ -347,22 +399,31 @@ def test_prediction_between_low_up(strategy: str, ensemble: bool) -> None:
...
 @pytest.mark.parametrize("method", ["plus", "minmax"])
 @pytest.mark.parametrize("cv", [-1, 2, 3, 5])
-def test_prediction_ensemble(method: str, cv: Union[LeaveOneOut, KFold]) -> None:
-    """Test that predictions differs when ensemble is True/False, but not prediction intervals."""
+def test_prediction_ensemble(
+    method: str,
+    cv: Union[LeaveOneOut, KFold]
+) -> None:
+    """
+    Test that predictions differs when ensemble is True/False,
```
Comment (vtaquet) [inline mapie/tests/test_estimators.py:407]: predictions differ

File: .flake8 [hunk 4]
```diff
@@ -1,2 +0,0 @@
-[flake8]
-max-line-length = 120
```
Comment (vtaquet) [inline .flake8:2]: Le fichier est enlevé afin de considérer la valeur par défaut de `max-line-length` ?

File: mapie/estimators.py (hunk 1 above)
Comment (gmartinonQM) [reply mapie/estimators.py:518]: Je ne pense pas, on peut mettre autant de variable que l'on veut sur une ligne. Flake8 ne s'en plaint pas en tout cas

File: mapie/estimators.py (hunk 2 above)
Comment (gmartinonQM) [reply mapie/estimators.py:574]: done

File: mapie/tests/test_estimators.py (hunk 3 above)
Comment (gmartinonQM) [reply mapie/tests/test_estimators.py:407]: done

File: .flake8 (hunk 4 above)
Comment (gmartinonQM) [reply .flake8:2]: yes

**Pull request merged**